packages/block-library/src/list/ordered-list-settings.js

```diff
@@ -24,6 +29,21 @@ const OrderedListSettings = ( { setAttributes, reversed, start } ) => (
 				value={ Number.isInteger( start ) ? start.toString( 10 ) : '' }
 				step="1"
 			/>
+			<SelectControl
+				__nextHasNoMarginBottom
+				label={ __( 'Numbering type' ) }
```

Comment:
Maybe this phrasing would be better?

```suggestion
				label={ __( 'Numbering style' ) }
```

Reply:
yeh, I like that better, thanks